index.bs

```diff
@@ -321,8 +323,13 @@ interface PictureInPictureWindow {
 };
 </pre>
 
-The {{PictureInPictureWindow}} is a simple proxy to get information such as
-dimensions about the current Picture-in-Picture window.
+The {{PictureInPictureWindow}} interface is a simple proxy to get information
+such as dimensions about the current <a>Picture-in-Picture window</a>.
+
+<dfn>Picture-in-Picture window</dfn> initial |state| is |closed|.
+
+When <a>Picture-in-Picture window</a> |state| is |closed|, its dimensions
+{{width}} and {{height}} SHOULD be set to 0.
```

Comment:
Shouldn't you define how `width` and `height` should work. Basically what they should return and for both mention the `closed` state as the case where 0 should be returned?